devo/sender/lookup.py

```diff
@@ -351,16 +352,16 @@ def list_to_headers(headers=None, key=None,
         return out
 
     @staticmethod
-    def field_to_str(field):
+    def field_to_str(field, escape_quotes=False):
         """
         Convert one value to STR, cleaning it
         :param field: field to clean
         :return:
         """
-        return ',%s' % Lookup.clean_field(field)
+        return ',%s' % Lookup.clean_field(field, escape_quotes)
 
     @staticmethod
-    def process_fields(fields=None, key_index=None):
+    def process_fields(fields=None, key_index=None, escape_quotes=False):
```

Comment:
This method is not only being used by [send_data_line](https://github.com/ErickLares/python-sdk/blob/7e2d3ebaad985a2a6685936cccae053f67518d4c/devo/sender/lookup.py#L84) but by [send_csv](https://github.com/ErickLares/python-sdk/blob/7e2d3ebaad985a2a6685936cccae053f67518d4c/devo/sender/lookup.py#L144) too. I think we should modify those calls as well.